app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,186 @@
+
+	<!-- Menu -->
+	<string name="action_settings">Paramètres</string>
+	<string name="action_support">Soutenir le développeur</string>
+	<string name="action_change_log">Historique des changements</string>
+	<string name="action_credits">Crédits</string>
+
+	<!-- Folder list -->
+	<string name="delete_button">Effacer</string>
+	<string name="back_folder">\u2191 Retour</string>
+
+	<string name="folder_list_empty_title">Bienvenue sur Phone Saver!</string>
+	<string name="folder_list_empty_body">Veuillez ajouter un lieu de sauvegarde avec le bouton ci-dessous.</string>
+
+	<string name="location_select_label">* Selectionner lieu *</string>
+
+	<!-- Folder select -->
+	<string name="title_activity_folder_select">Selectionner Fichier</string>
+	<string name="navigation_here">Ici</string>
+	<string name="navigation_read_only">Lecture uniquement</string>
+
+	<!-- Saving messages -->
+	<string name="toast_save_successful">Sauvegardé avec succès.</string>
+	<string name="toast_save_in_progress">Sauvegarde en cours…</string>
+	<string name="toast_save_failed">Sauvegarde échouée.</string>
+	<string name="toast_save_init_error">Erreur de chargement des lieus de sauvegarde.</string>
+	<string name="toast_save_init_no_locations">Veuillez ouvrir l\'application pour ajouter des lieus de sauvegarde.</string>
+	<string name="toast_save_file_exists_overwrite">Ecrasement du fichier présent à cet endroit…</string>
+	<string name="toast_save_file_exists_fail">Impossible de sauvegarder, ce fichier existe déjà.</string>
+	<string name="toast_save_file_exists_self_abort">Abandon! Il semblerait que vous remplacez ce fichier par lui-même.</string>
+	<string name="toast_save_file_exists_self_continue">Attention! Il semblerait que vous
+		remplacez ce fichier par lui-même. Comme permi par le comportement de sauvegarde, écrasement du fichier…</string>
+	<string name="downloader_description">Sauvegarde de %1$s</string>
+
+	<!-- Debug / Error -->
+	<string name="not_supported_content_type">Type de contenu inacceptable</string>
+	<string name="not_supported" tools:keep="@string/not_supported">Veuillez &lt;a href="%1$s">faire une demande d\'ajout de fonctionnalité&lt;/a> avec ce
+		type de contenu afin que &author; puisse ajouter un support officiel. &lt;br/>&lt;br/>Vous pouvez également activer le paramètre \"sauvegarde forcée\"
+		setting pour forcer &appname; à essayer de le sauvegarder.</string>
+
+	<!-- Loading -->
+	<string name="loading_title">Test du support…</string>
+	<string name="loading_body">Veuillez patienter tandis que je test le téléchargement du support…</string>
+
+	<!-- Credits -->
+	<string name="credits_version">v%1$s</string>
+	<string name="title_activity_credits">A propos de &appname;</string>
+	<string name="logo_description">Logo</string>
+	<string name="credits_creator">par <a href="https://michael.standen.link">&author;</a></string>
+	<string name="credits_content1">Phone Saver est une simple application pour sauvegarder
```

Comment:
Could you please use `&appname;` here instead of `Phone Saver`

Reply:
Will do